alea/include/alps/alea/autocorr.hpp

```diff
@@ -66,7 +66,7 @@ template <typename T>
 class autocorr_acc
 {
 public:
-    typedef T value_type;
+    using value_type = T;
     typedef typename bind<circular_var, T>::var_type var_type;
     typedef var_acc<T, circular_var> level_acc_type;
```

Comment:
Why this and not the other typedef's here?

Reply:
No real reason. `mean_acc` was missing the `typedef` so I put a `using` expression and then wanted to make it consistent across all the accumulators. I can update rest of the `typedefs` if you want or switch `value_type` back.  I overall prefer the `using` syntax but it doesn't matter too much.

Reply:
Yeah, let's do that then.

Reply:
switch to `using` or back to `typedef`?

Reply:
switch to `using` -- I feel like this is the way the kids call it these days :-)